src/transform/classify.rs

```diff
@@ -0,0 +1,69 @@
+/// Transforms for classifying (tagging and categorizing) events.
+///
+/// Based on code in aw_research: https://github.com/ActivityWatch/aw-research/blob/master/aw_research/classify.py
+
+use regex::Regex;
+use serde_json;
+
+use std::collections::HashSet;
+use crate::models::Event;
+
+pub fn classify(mut events: Vec<Event>, classes: Vec<(String, Regex)>) -> Vec<Event> {
```

Comment:
Maybe I'm missing something, but isn't it unnecessary to take ownership of the classes here?

Reply:
It is, I'll fix it.